extras/rya.streams/api/src/main/java/org/apache/rya/streams/api/queries/QueryChangeLogListener.java

```diff
@@ -36,6 +40,7 @@
      * on the repository.
      *
      * @param queryChangeEvent - The event that occurred. (not null)
+     * @param newQueryState - The new state of the query after the query change event. (not null)
```

Comment:
Indicate when this is not present. I assume it's whenever the change is a DELETE.

Reply:
done